packages/block-editor/src/components/provider/block-refs-provider.js

```diff
@@ -3,7 +3,10 @@
  */
 import { createContext, useMemo } from '@wordpress/element';
 
-export const BlockRefs = createContext();
+export const BlockRefs = createContext( {
+	refs: new Map(),
+	callbacks: new Map(),
+} );
```

Comment:
Why this was needed?

Reply:
Without it the "Iframe" with a separate GlobalStyles context was breaking because somewhere down the react tree there's a consumer for this context that assumes it was initialized properly, but the default value (if not provider is explicitly added) was not.